.github/workflows/ci-tests.yaml

```diff
@@ -74,3 +74,27 @@ jobs:
         with:
           user: __token__
           password: ${{ secrets.PYPI_API_TOKEN }}
+
+  browser-save-image-test:
+    name: "Test for saving table image with browser"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: browser-actions/setup-firefox@v1
+      - run: firefox --version
+      - uses: actions/checkout@v4
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.12"
+      - name: Install dependencies
+        run: |
+          pip install -e '.[extra]'
```

Comment:
I would just run `pip install pandas` for now after this, since we need it for the datasets